### PR TITLE
Add Thai translation

### PR DESCRIPTION
**Pull request by kerlos**

### CODE REVIEW

File: resources/config.yml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 ---
 # The language that should be used in the plugin
-# Available languages (ISO639-3): 
-# ara, cat, ces, deu, ell, eng, esp, fre, hin, ita, jpn, kor, lit, mas, nld, nor, pol, por, rom, slo, spa, swe, swh, tgl, zho
+# Available languages (ISO639-3):
+# ara, cat, ces, deu, ell, eng, esp, fre, hin, ita, jpn, kor, lit, mas, nld, nor, pol, por, rom, slo, spa, swe, swh, tgl, tha, zho
```
Comment (inxomnyaa) [inline resources/config.yml:4]: Oh indeed, i need to update this after the prs get merged

File: resources/lang/tha.ini [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+ui.brush.options.blocks = "บล็อก"
+ui.brush.options.blocks.placeholder = "ตัวอย่าง: 1:1,2,tnt,log:12"
+ui.brush.options.diameter = "เส้นผ่าศูนย์กลาง"
+ui.brush.options.width = "กว้าง"
+ui.brush.options.height = "สูง"
+ui.brush.options.depth = "ลึก"
+ui.brush.options.flags = "เพิ่ม flags?"
+; language
+ui.language.title = "เลือกภาษา"
+ui.language.label = "ตั้งค่าภาษาของเซสชั่นของคุณ ถ้าไม่มีภาษาของคุณ คุณสามารถช่วยแปล plugin ได้ที่ GitHub!"
```
Comment (inxomnyaa) [inline resources/lang/tha.ini]: are this specific terms that cannot be translated?

Comment (kerlos) [reply resources/lang/tha.ini]: I've updated translation.

**Review by inxomnyaa — APPROVED** [review]

**Comment by Aericio** [comment]
Are we waiting for Hacktoberfest to complete before merging?

**Comment by inxomnyaa** [comment]
@Aericio Yes - as stated in #130 i will merge around 30th.

**Comment by inxomnyaa** [comment]
Oh. Looks like there was another thai PR that i merged.. now there are conflicts

**Comment by inxomnyaa** [comment]
Reverted the other PR and chose this one instead

**Pull request merged**